.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+node_js:
+  - stable
+  - 8.0.0
+
```

Comment:
Use only one version of node. i.e. remove line 3 and edit line 4 to `"8"`

Reply:
Okay then.